test/cleanup.test.js

```diff
@@ -1,187 +1,284 @@
 'use strict';
 
 /* eslint-disable no-console */
-process.argv[4] = '.';
+
+const child_process = require('child_process');
 const AWS = require('@mapbox/mock-aws-sdk-js');
 const sinon = require('sinon');
 const test = require('tape');
-const file = require(`${__dirname}/../scripts/cleanup`);
+const cleanup = require(`${__dirname}/../scripts/cleanup`);
+
 const region = 'us-east-1';
 const repo = 'some-repo';
 const token = 'sometoken';
-const error = 'some error';
-const success = 'some success';
+const tmpdir = '/path/to/cloned/repo';
+
 const imagesNoToken = require(`${__dirname}/fixtures/imagesNoToken.test.json`);
 const imagesToken = require(`${__dirname}/fixtures/imagesToken.test.json`);
 const imageDetails = imagesNoToken.imageDetails.concat(imagesToken.imageDetails);
-const imageIds = [{ imageDigest: imagesNoToken.imageDetails[1].imageDigest }];
-
-test('mockSpawn', (t) => {
-  let gitMergeCommitStdout;
-  let spawns = (command, args) => {
-    switch(command) {
-    case 'git':
-      if(args[3] === 'cccccccccccccccccccccccccccccccccccccccc' || args[3] === 'eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee') {
-        t.equals(args[0], `--git-dir=${process.argv[4]}/.git`);
```

Comment:
Furthermore -- what is `t` in this situation? It is in the closure for **this** test -- not the subsequent test that you run later which would fall into this block.